### PR TITLE
add card data to site views

### PR DESCRIPTION
**Pull request by antonpiatek**

### CODE REVIEW

File: app.coffee [hunk 1]
```diff
@@ -32,6 +32,7 @@ dashboard = require('./routes/dashboard')(app)
 subscription = require('./routes/subscription')(app)
 admin = require('./routes/admin')(app)
 auth = require './routes/auth'
+adminapi = require('./routes/adminapi')(app)
```
Comment (antonpiatek) [inline app.coffee:35]: Let me know if this should simply be rolled into the user class instead

File: app.coffee [hunk 2]
```diff
@@ -172,6 +173,9 @@ app.all '/admin/money', admin.money
 app.all '/admin/reminders', admin.reminders
 app.all '/admin/emails', admin.emails
 
+# restricted API-like
+app.get '/adminapi/cards', adminapi.cards
```
Comment (antonpiatek) [inline app.coffee:177]: It seems to me we may want a whole path/folder for putting future apis, though we could debate if we want admin in the name

File: views/dashboard.jade [hunk 3]
```diff
@@ -20,3 +20,11 @@ block content
     p.text-success You're paid up until 
       strong= loggedInUser.paidUntil.toString()
       |  - thanks!
+  h4 RFID
+  if loggedInUser.getData().cards
+    if loggedInUser.getData().cards.length == 1
+      p #{loggedInUser.getData().cards.length} card registered for access
```
Comment (antonpiatek) [inline views/dashboard.jade:26]: As the ID of the card won't be much use to the user, I don't think there is value in displaying it. If users want to name their cards that would make more sense, but waiting for someone to actually want that first

**Comment by benjie** [comment]
@antonpiatek Happy with the changes, made a few minor changes (haven't tested them though), hopefully they'll work... The API URL will probably change (along with everything else) with the new members area. I'm not aiming to maintain backwards compatibility with this software.

**Comment by antonpiatek** [comment]
No problem with the uris changing at all. Thanks for merging

**Comment by benjie** [comment]
Sorry it took so long :(

**Pull request merged**